### ISSUE TITLE
Getting execvp errors when running binaries on the same machine

### ISSUE THREAD

**Issue by AleXoundOS** [matthewbauer/nix-bundle#62]
```console
$ git clone https://github.com/matthewbauer/nix-bundle.git
$ cd nix-bundle
$ ./nix-bundle.sh -f channel:nixos-19.09 hello --bin /bin/hello
[1 copied (0.2 MiB), 13.3 MiB DL]
(with import ./default.nix {}; nix-bootstrap { target = "/nix/store/4w99qz14nsahk0s798a5rw5l7qk1zwwf-hello-2.10"; extraTargets = [ ]; run = "/bin/hello"; })
these derivations will be built:
  /nix/store/ix3s9xy4lb79g3h21m48xhg8z93kj251-startup.drv
  /nix/store/r30gwpi80v5p74pr82xwglckfmsrb3wd-maketar.drv
  /nix/store/qjb83jrdckr9yrm2xzjv5w5a7y1i7ydf-arx.drv
building '/nix/store/ix3s9xy4lb79g3h21m48xhg8z93kj251-startup.drv'...
building '/nix/store/r30gwpi80v5p74pr82xwglckfmsrb3wd-maketar.drv'...
tar: Removing leading `/' from member names
building '/nix/store/qjb83jrdckr9yrm2xzjv5w5a7y1i7ydf-arx.drv'...
Nix bundle created at hello.
$ ./hello 
execvp(/nix/store/4w99qz14nsahk0s798a5rw5l7qk1zwwf-hello-2.10/bin/hello): No such file or directory
```

Running on NixOS 19.09.1320.4ad6f1404a8.

Am I doing something wrong? _README is somewhat outdated_.

**Comment by matthewbauer** [comment]
This should be fixed with 2a8b914e1b14706a0d3bbc5f8f43af624ef48183

**Comment by matthewbauer** [comment]
This is very weird!
